types/backblaze-b2/index.d.ts

```diff
@@ -0,0 +1,168 @@
+// Type definitions for backblaze-b2 1.5
+// Project: https://github.com/yakovkhalinsky/backblaze-b2
+// Definitions by: Rohith Bhaskaran <https://github.com/rohithb>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+export as namespace BackBlazeB2;
+
+export = BackBlazeB2;
+
+interface B2InitOptions {
+    applicationKeyId: string;
+    applicationKey: string;
+    axios?: Record<string, any>;
+    retry?: Record<string, any>;
+}
+
+interface CommonArgs {
+    axios?: Record<string, any>;
+    axiosOverride?: Record<string, any>;
+}
+
+interface StandardApiResponse {
+    status: string;
+    statusText: string;
+    headers: any;
+    config: any;
+    request: any;
+    data: any;
+}
+type BucketType = 'allPublic' | 'allPrivate';
+
+interface CreateBucketOpts extends CommonArgs {
+    bucketName: string;
+    bucketType: BucketType;
+}
+interface GetBucketOpts extends CommonArgs {
+    bucketName: string;
+    bucketId?: string;
+}
+interface UpdateBucketOpts extends CommonArgs {
+    bucketId: string;
+    bucketType: BucketType;
+}
+interface UploadProgressFn {
+    (event: any): void;
+}
+
+interface UploadFileOpts extends CommonArgs {
+    uploadUrl: string;
+    uploadAuthToken: string;
+    fileName: string;
+    data: Buffer;
+    /**
+     * data length
+     * @default  data.byteLength or data.length
+     */
+    contentLength?: number;
+    /**
+     * mime type
+     * @default 'b2/x-auto'
+     */
+    mime?: string;
+    /**
+     * data hash
+     * @default sha1(data)
+     */
+    hash?: string;
+    // optional info headers, prepended with X-Bz-Info- when sent, throws error if more than 10 keys set
+    // valid characters should be a-z, A-Z and '-', all other characters will cause an error to be thrown
```

Comment:
```suggestion
```
seeems the diff snippet was not applied correctly, those inline comments are no longer requried I think